### ISSUE TITLE
$options on Laminas\ServiceManager\Factory\FactoryInterface

### ISSUE THREAD

**Issue by fezfez** [Slamdunk/phpstan-laminas-framework#26]
Hello,

I got this error on every class that implements `Laminas\ServiceManager\Factory\FactoryInterface`.

```
Method MyFactory::__invoke() has parameter $options with no value type specified in iterable type array.  
         💡 See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type 
```

I think it should be resolved by this extension.

Thanks for you work !

**Comment by Slamdunk** [comment]
Heh, the correct place to fix this is https://github.com/laminas/laminas-servicemanager/blob/1a32d6811fcbf54fd59f7a4763acd52fca8a7c97/src/Factory/FactoryInterface.php#L26, not this library

**Comment by fezfez** [comment]
seem a bit duplicate because it's already declared in method `?array $options = null`, no ?

**Comment by Slamdunk** [comment]
No, this is the diff that would fix the warning:

```diff
 interface FactoryInterface
 {
     /**
      * Create an object
      *
      * @param  string             $requestedName
-     * @param  null|array         $options
+     * @param  null|array<mixed>  $options
      * @return object
      * @throws ServiceNotFoundException If unable to resolve the service.
      * @throws ServiceNotCreatedException If an exception is raised when creating a service.
      * @throws ContainerException If any other error occurs.
      */
     public function __invoke(ContainerInterface $container, $requestedName, ?array $options = null);
 }
```
